applications/main/nfc/scenes/nfc_scene_mfkey_nonces_info.c

```diff
@@ -16,14 +16,14 @@ void nfc_scene_mfkey_nonces_info_on_enter(void* context) {
 
     uint16_t nonces_saved = mfkey32_get_auth_sectors(temp_str);
     widget_add_text_scroll_element(nfc->widget, 0, 22, 128, 42, string_get_cstr(temp_str));
-    string_printf(temp_str, "Nonces saved %d", nonces_saved);
+    string_printf(temp_str, "Nonce pairs saved %d", nonces_saved);
```

Comment:
I think that "Saved %d nonce pairs" would fit better here

Reply:
Personally I prefer "description - value" format. Also it complies with Miro